pytition/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -1305,7 +1306,7 @@ msgstr "Voulez-vous vraiment transférer la pétition suivante ?"
 
 #: pytition/petition/templates/petition/transfer_petition.html:54
 msgid "It will be transfered to:"
-msgstr "Elle va être transférée à : "
+msgstr "Elle va être transférée à :"
```

Comment:
Here I guess the original translation was correct, in French after ':' you need an unbreakable space.